doc/Type/Cool.pod6

```diff
@@ -707,7 +707,7 @@ number of characters in the string. Please note that on the JVM, you currently g
 codepoints instead of graphemes.
 
     say 'møp'.chars;    # OUTPUT: «3␤»
-    say 'ã̷̠̬̊'.chars;     # OUTPUT: «1␤»
+    say 'ã̷̠̬̊'.chars;      # OUTPUT: «1␤»
```

Comment:
I can fix these lines, but there are a few lines that will still show up as having been changed by the script, I'm guessing because of line endings. I haven't found out exactly why, but what changed isn't visible from Vim. Is it alright to allow those lines to stay changed?